### ISSUE TITLE
[Inputs.json]: Talkback and Keyboard focus are not in sync in DatePicker under Due Date.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#7038]
https://microsoft.visualstudio.com/OS/_workitems/edit/38004752

**Comment by anna-dingler** [comment]
Hi @MohitTyagi13,

We utilize the Android's `DatePickerDialog` to create this control, so we are unable to modify the code.

I have investigated updating to Android 13 and updating `androidx.appcompat:appcompat`; however, there does not seem to be accessibility fixes that impact will impact our `Input.Date` control.

Please let me know if there is anything else I can do here. Thanks!

**Comment by Raisul123** [comment]
Varified on Adaptive Card, Teams app and Android Native Calendar the issue seems to be similar through out. However, with Talk Back keyboard navigation [Alt + arrow keys] user can navigaate on to the Calendar dates. 

Hence the severity can be reduced as keyboard users including Braille Display users aren't impacted.

**Comment by Raisul123** [comment]
Issue is still repro.
Reactivating since this bug depend on Android.

**Comment by vagpt** [comment]
@jwoo-msft,

Hope you are doing great.

Could you please provide us the codepen of this bug?

**Comment by vagpt** [comment]
Hi @jwoo-msft , @JeanRoca 

Hope you are doing great.

Could you please provide us the codepen of this bug?
